src/python.rs

```diff
@@ -11,25 +11,33 @@ use RefVersion::*;
 create_exception!(dbt_extractor, ExtractionError, PyException);
 
 // Can't export ConfigVal directly: https://github.com/PyO3/pyo3/issues/417
-fn convert_config(py: Python, v: ConfigVal) -> PyObject {
+fn convert_config(py: Python, v: ConfigVal) -> Py<PyAny> {
     match v {
-        ConfigVal::StringC(s) => s.to_object(py),
-        ConfigVal::BoolC(b) => b.to_object(py),
+        ConfigVal::StringC(s) => s.into_pyobject(py).unwrap().into(),
+        #[allow(deprecated)]
+        ConfigVal::BoolC(b) => {
+            let py_obj = b.into_py(py);
+            Py::from(py_obj)
+        },
         ConfigVal::ListC(v) => v
             .into_iter()
             .map(|x| convert_config(py, x))
-            .collect::<Vec<PyObject>>()
-            .to_object(py),
+            .collect::<Vec<Py<PyAny>>>()
+            .into_pyobject(py)
+            .unwrap()
```

Comment:
nit: consider `.expect("reason this should work")`

Reply:
Reason is `unwarp` is considered prototyping code(since the error case is not dealt with.